src/GitHubApps/src/DarcBot/DarcBot.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <AzureFunctionsVersion>v2</AzureFunctionsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="Settings.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\..\..\Shared\Microsoft.DotNet.Kusto\CsvWriterExtensions.cs" Link="CsvWriterExtensions.cs" />
```

Comment:
Just reference the assembly. Don't reference the files themselves.

Reply:
Azure Functions wasn't happy with this method and refused to reference the assembly, I'm opting to leave this as-is for now. :/

Reply:
What do you mean "wasn't happy"?  Do we need to fix the framework version? Source level sharing isn't really ok.  It's a huge source of very hard to diagnose bugs, and there shouldn't be any reason you can't reference this assembly.  We reference a bunch of other assemblies... what's special/wrong about this one?

Reply:
When running, the Azure function blows up with

```
Could not load file or assembly 'Microsoft.DotNet.Kusto, Version=42.42.42.42, Culture=neutral, PublicKeyToken=null'.
```

Reply:
Is there anything more? Any further details?  Maybe @alexperovich can help? I do not like Azure functions, because they have a lot of implicit and hard to debug problems like this. :-|

Reply:
That's all there is

Reply:
@alexperovich thoughts?  I thought the nonsense with loading assemblies was removed for Azure Functions 2.

I'm trying to include `Microsoft.DotNet.Kusto` as a ProjectReference and it's blowing up with the above error.  Other ProjectReference's seem to be fine. Forums aren't particularly useful because there are so many possible reasons.

Reply:
I think Azure Functions have a problem running 64 bit processes (by default), I'm investigating.

Reply:
Yep, that appears to be the problem.  The default is to run as 32 bit.  I found an (old) [article](https://benperk.github.io/msdn/2017/2017-02-azure-functions-on-64-bit.html) that talked about this.  Fortunately, the new Azure Portal UI has a toggle that enables 64 bit.  That, plus restarting the function, seems to have resolved the problem.